recipes/lammps/2024/gh200/environments.yaml

```diff
@@ -11,16 +11,57 @@ lammps:
   - cmake
   - cuda@12
   - kokkos@4.3.00 +wrapper +cuda cuda_arch=90
-  - mpi
-  - cray-mpich
-  - lammps@20240207.1 +cuda cuda_arch=90 +python +kokkos
+  # lammps + kokkos package
+  - lammps +kokkos +cuda cuda_arch=90 +python +extra-dump
   variants:
   - +mpi
   - +cuda
   - cuda_arch=90
   views:
-    develop:
-      link: roots
+    develop-kokkos:
+      link: all
```

Comment:
This is explicitly discouraged in the UENV documentation (https://eth-cscs.github.io/alps-uenv/pkg-application-tutorial/#environments); is there a particular reason why `link: roots` does not work here?

Reply:
link: all is needed here because we enable the +python option for LAMMPS, and currently spack will only move all python lib dependencies into the site-packages folder if "all" is used.